tests/integrations/django/asgi/test_asgi.py

```diff
@@ -18,8 +18,11 @@
 
 @pytest.mark.parametrize("application", APPS)
 @pytest.mark.asyncio
-async def test_basic(sentry_init, capture_events, application):
+async def test_basic(sentry_init, capture_events, application, request):
     sentry_init(integrations=[DjangoIntegration()], send_default_pii=True)
+    Hub.main.bind_client(Hub.current.client)
+    request.addfinalizer(lambda: Hub.main.bind_client(None))
```

Comment:
Why do we need this now?

Reply:
I had an argument one week ago for this, but can't remember it. I tried removing those lines and it works just fine. Removed.